### ISSUE TITLE
next challenge page code frame visibility

### ISSUE THREAD

**Issue by Emil-Valentin-Toma** [freeCodeCamp/freeCodeCamp#35836]
Hi,
I came across a little bug. 
As soon as I got a passed test frame and then Ctrl+Enter, the next challenge page, although it loads completely, shifts up one line in my preferred browser, Firefox 67.0b7 (64-bit) (screenshot enclosed). I tested the same in Chrome 73.0.3683.103 (Official Build) (64-bit) and the issue is not present. Also, the issue is not present at all while looping back or forward to throughout the pages that I passed the test, or if I land on the challenge page from curriculum page.

It doesn't seem major, why I posted? Lately, being used with the first line containing a comment, I didn't scroll the page, assuming that there was a comment. Therefore I wrote a function line in duplicate, without being aware it was already written for me, on the first line. It took me a while to figure that out :) Ultimately, the extra curly brace gave me the clue. As such, I decided to let you know about the little bug.

The particular page I encountered the issue is this:
[https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/stand-in-line](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/stand-in-line)

Can I can contribute to the fixing of this issue please?

Thank you!

![emil_toma_pageShiftedUpOneLine_13Apr2019](https://user-images.githubusercontent.com/25068526/56076981-8b1b9e80-5dce-11e9-8b9a-bc34ed3c2a62.PNG)

**Comment by RandellDawson** [comment]
@Emil-Valentin-Toma Can you test this on our [beta site](https://freecodecamp.rocks).  The beta site has many fixes in place and might have already been taken care of.  Let us know.

Thanks.

**Comment by Emil-Valentin-Toma** [comment]
Hi @RandellDawson,
I replicated the test and the error is not present on the beta site. The code frame visibility is normal, starting with the first line (screenshot enclosed). Therefore the bug was fixed for this particular setting.
I didn't check other than the two pages necessary for replicating the test sequence. I'd be glad to perform other tests if you wish, as per instructions.

It looks like you going to store the user's solution on the challenge page? That would be very appreciated!

Best regards.

![pageShiftedUpOneLine_fixed_14Apr2019 -tested by emil_toma Firefox 67 0b7 64-bit](https://user-images.githubusercontent.com/25068526/56088872-253c1f00-5e81-11e9-9b39-1af431312899.PNG)

**Comment by RandellDawson** [comment]
I am going to close this issue for now, since it can not be replicated on the beta site which will soon be moved to production.  Feel free to request it be reopened if you discover any problems after further testing of the beta site.

> It looks like you going to store the user's solution on the challenge page? That would be very appreciated!

Not sure what you mean here.  What gives you this indication?

**Comment by Emil-Valentin-Toma** [comment]
*"It looks like you going to store the user's solution on the challenge
page? That would be very appreciated!*

*Not sure what you mean here. What gives you this indication?"*

           On the actual site, as soon as I navigate further to the next
challenge, all the data I wrote (the solution) for the passed challenge is
lost. Should I need to refresh memory - no chance. If I accessed a solved
challenge it will show a bleak fresh challenge. Many times I had to do
challenges all over again in order to refresh notions. Good for learning.

However, I noticed that on beta site the solution to challenge is stored
(saved). I was able to access again the solution I wrote, regardless of how
many pages I navigated. That's all, long waited and fixed. Hopefully you'll
preserve this feature on the released version.
